CatAnalyzer/plugins/h2muAnalyzer.cc

```diff
@@ -118,17 +118,17 @@ h2muAnalyzer::h2muAnalyzer(const edm::ParameterSet& iConfig)
   ttree_->Branch("gen_mu1_eta", &b_gen_mu1_eta, "gen_mu1_eta/F");
   ttree_->Branch("gen_mu1_phi", &b_gen_mu1_phi, "gen_mu1_phi/F");
   ttree_->Branch("gen_mu1_ptRes", &b_gen_mu1_ptRes, "gen_mu1_ptRes/F");
-  ttree_->Branch("gen_mu1_isLoose", &b_gen_mu1_isLoose, "gen_mu1_isLoose/F");
-  ttree_->Branch("gen_mu1_isMedium", &b_gen_mu1_isMedium, "gen_mu1_isMedium/F");
-  ttree_->Branch("gen_mu1_isTight", &b_gen_mu1_isTight, "gen_mu1_isTight/F");
+  ttree_->Branch("gen_mu1_isLoose", &b_gen_mu1_isLoose, "gen_mu1_isLoose/B");
+  ttree_->Branch("gen_mu1_isMedium", &b_gen_mu1_isMedium, "gen_mu1_isMedium/B");
+  ttree_->Branch("gen_mu1_isTight", &b_gen_mu1_isTight, "gen_mu1_isTight/B");
```

Comment:
"/O" to store boolean.

    C : a character string terminated by the 0 character
    B : an 8 bit signed integer (Char_t)
    b : an 8 bit unsigned integer (UChar_t)
    S : a 16 bit signed integer (Short_t)
    s : a 16 bit unsigned integer (UShort_t)
    I : a 32 bit signed integer (Int_t)
    i : a 32 bit unsigned integer (UInt_t)
    F : a 32 bit floating point (Float_t)
    D : a 64 bit floating point (Double_t)
    L : a 64 bit signed integer (Long64_t)
    l : a 64 bit unsigned integer (ULong64_t)
    O : [the letter 'o', not a zero] a boolean (Bool_t)